language-support/hs/bindings/BUILD.bazel

```diff
@@ -36,3 +37,36 @@ da_haskell_binary(
         "//language-support/hs/bindings:hs-ledger",
     ],
 )
+
+daml_compile(
+    name = "quickstart",
+    srcs = glob(["test/daml/quickstart/*.daml"]),
+    main_src = "test/daml/quickstart/Main.daml",
+    target = "1.1",
```

Comment:
Why not 1.3?

Reply:
You could also just not specify it in which case the default of the compiler will be used which has the advantage that we notice breakages when we upgrade that.

Reply:
I removed the version